spec/models/sighting_spec.rb

```diff
@@ -10,4 +10,17 @@
     it { should validate_presence_of(:title) }
     it { should validate_presence_of(:location) }
   end
+
+  describe 'class methods' do
+    describe 'location_search' do
+      it 'returns array of sightings by location' do
```

Comment:
This test looks good. I think you should also add tests for the edge case when there is an empty query string and also test the case insensitive functionality you added to the method.